awx/main/models/mixins.py

```diff
@@ -483,3 +489,138 @@ def get_active_jobs(self):
             raise RuntimeError("Programmer error. Expected _get_active_jobs() to return a QuerySet.")
 
         return [dict(id=t[0], type=mapping[t[1]]) for t in jobs.values_list('id', 'polymorphic_ctype_id')]
+
+
+class WebhookTemplateMixin(models.Model):
+    class Meta:
+        abstract = True
+
+    SERVICES = [
+        ('github', "Github"),
```

Comment:
I think technically speaking, the names are GitHub and GitLab

Reply:
So it is.  How ugly.